### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by chrisrohr**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    <version>0.3.0</version>
+  </parent>
+
+  <artifactId>dropwizard-service-utilities</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>
+    Set of utilities to aide in setting up Dropwizard services
```
Comment (sleberknight) [inline pom.xml:18]: That would be "aid" 😜

Comment (sleberknight) [reply pom.xml:18]: (an aide is an assistant to an important person)

Comment (chrisrohr) [reply pom.xml:18]: haha yes thank you

**Review by sleberknight — APPROVED** [review]
Just fix the "aide" thing...😂

**Pull request merged**